Automattic/wp-calypso
# Jetpack Cloud: Update Manifest.json

#### Changes proposed in this Pull Request

* Use Jetpack.com colors/names in Manifest.json

<img width="355" alt="Screen Shot 2020-05-20 at 5 44 28 PM" src="https://user-images.githubusercontent.com/2810519/82511353-ea63dc00-9ac1-11ea-853d-3998f34ec9f4.png">

#### Testing instructions

1. Boot Jetpack Cloud and click the Install button in Chrome

<img width="1229" alt="Screen Shot 2020-05-20 at 5 44 11 PM" src="https://user-images.githubusercontent.com/2810519/82511298-c902f000-9ac1-11ea-8442-6570181f3d62.png">

2. Verify the values are for "Jetpack.com" and not "WordPress.com"
3. Boot Calypso and verify the install is for WordPress.com
<img width="363" alt="Screen Shot 2020-05-20 at 5 46 21 PM" src="https://user-images.githubusercontent.com/2810519/82511460-42024780-9ac2-11ea-9a93-edc258887f16.png">
